test/parallel/test-global-setters.js

```diff
@@ -1,4 +1,10 @@
-/* eslint-disable strict */
+// When setters and getters were added for global.process and global.Buffer to
+// create a deprecation path for them in ESM, this test was added to make sure
```

Comment:
It could be worth noting at this point it's no longer about creating a deprecation path since that ship has likely sailed short of a very big breaking change, but that these getters and setters may be used to attenuate access if we gain the ability to gate the global object by module accessor in future somehow.

Reply:
I wasn't really sure how to word this for the comment, but if someone wants to open a `fast-track` PR to update the comment to reflect this, please do! (I kind of wonder if it makes sense to remove the getters/setters at this point. But I'm not terribly informed on all the details/context, so that might be misguided of me.)